content/en/guides/deployment/deployment-amazon-web-services.md

```diff
@@ -0,0 +1,405 @@
+---
+title: 'How to Deploy on Amazon Web Services'
+description: Static Hosting on AWS with S3 Amplify and CloudFront
+menu: Deploy on Amazon Web Services
+category: deployment
+position: 201
+---
+
+AWS stands for Amazon Web Services.  
+S3 is their static storage which can be configured for Static Site Hosting. CloudFront is their CDN (content delivery network)
+
+## AWS w/ the Amplify Console
+
+Hosting a **static generated** Nuxt app on AWS w/ the Amplify Console is powerful and cheap.
+
+First, push your Nuxt app to the Git provider of your choice. Then, visit the [Amplify Console](https://console.aws.amazon.com/amplify/home). Click the **GET STARTED** button under the **Deploy** header if you haven't used Amplify Hosting before, otherwise click the **Connect App** button.
+
+### From your existing code
+
+On the "From your existing code" page, select your Git provider and click **Continue**.
+
+### Add repository branch
+
+On the "Add repository branch" page, select your repository and the branch you want to deploy. Then, click **Next**.
+
+### Configure build settings
+
+On the "Configure build settings" page, click the `Edit` button under the "Build and test settings". Change the following:
+
+1. Set the **build** commands to `npm run generate`.
+2. Set the `baseDirectory` location to be `dist`.
+
+The settings should look like this once you are done editing them:
+
+```yml
+version: 1
+frontend:
+  phases:
+    preBuild:
+      commands:
+        - yarn install
+    build:
+      commands:
+        - npm run generate
+  artifacts:
+    # IMPORTANT - Please verify your build output directory
+    baseDirectory: dist
+    files:
+      - '**/*'
+  cache:
+    paths:
+      - node_modules/**/*
+```
+
+Then, click **Save** and **Next**.
+
+### Review
+
+On the review page, click **Save and deploy**.
+
+Then, your application will deploy. This may take a few minutes.
+
+Once `Provision`, `Build`, `Deploy`, and `Verify` are green, click on the URL that the Amplify Console provides to view your site.
+
+## AWS w/ S3 + CloudFront
```

Comment:
I'm wondering why all this manual setup is picked instead of using serverless framework?

Reply:
no idea. it could be old and not useful anymore. Maybe @aspittel can help us on this one? We have merged your amplify doc with the cloudfront one so would be good to know that both are ok. Thanks Ali

Reply:
Indeed some of guides seem outdated which i figured out later that this PR is a refactor. We might keep as TODO in related issue to improve later if it helps :)

Reply:
Agree to review this one later one as a separate issue.

Reply:
Hey! Sorry, just now seeing this -- GitHub notifications are tough to filter through! The Amplify deployment is the preferred and much easier approach. The other one will work, but I would lean towards just keeping Amplify.